pyfolio/perf_attrib.py

```diff
@@ -97,6 +98,33 @@ def perf_attrib(returns, positions, factor_returns, factor_loadings,
             2017-01-01  0.249087  0.935925        1.185012          1.185012
             2017-01-02 -0.003194 -0.400786       -0.403980         -0.403980
     """
+
+    missing_stocks = positions.columns.difference(
+        (factor_loadings.index.levels[1]))
```

Comment:
Extra set of parentheses here I believe (also I generally prefer to put the closing paren on the next line when formatting like this)

Reply:
fixed